PointOfInteraction/SmartSpot/doc/spec.md

```diff
@@ -0,0 +1,70 @@
+# SmartSpot
+
+## Description
+
+Smart Spots are devices which provide the technology to allow users to interact (suggestions mailbox, co-creation, …) or to obtain extra information (tourism, infotainment, …). The data model contains resources to configure the interaction service such as the broadcasted URL (probably will be a shortened url), the interval between broadcasts (frequency), the availability of the service, transmission power in function of the area to cover, etc.
```

Comment:
the technology which allows users to get access to smart points of interaction so that they can obtain extra information, suggestions or generate new content.